support-frontend/assets/pages/contributions-landing/components/ContributionTextInput.jsx

```diff
@@ -12,7 +12,7 @@ type PropTypes = {|
   id: string,
   name: string,
   label: string,
-  placeholder: boolean | string,
+  placeholder: boolean | string | null,
```

Comment:
Is boolean still a valid value for this?

Reply:
no - just removed it!